tests/Custom/InstantiationTest.php

```diff
@@ -3,8 +3,9 @@
 namespace Behat\Mink\Tests\Driver\Custom;
 
 use Behat\Mink\Driver\ZombieDriver;
+use Behat\Mink\Tests\Driver\TestCase;
 
-class InstantiationTest extends \PHPUnit_Framework_TestCase
+class InstantiationTest extends TestCase
```

Comment:
extend `PHPUnit\Framework\TestCase` instead. There is no reason to extend the driver testcase here as it does not rely on its features.

Reply:
I can't extends `PHPUnit\Framework\TestCase. It requries newer verison of phpunit, which won't be installed on older version of PHP, can I?

Reply:
see https://github.com/minkphp/MinkBrowserKitDriver/blob/4a0b453c8da1f6035f526a8532c293bba742d00d/.travis.yml#L31-L33 for the way we deal with it in other drivers.